_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -88,7 +92,7 @@ This page has `lang` attribute value of `en` (English), which matches the [defau
 
 #### Passed Example 2
 
-This page has `lang` attribute value of `en` (English), which matches the [default language of the page][default page language]. The default langage is English because all but a few words are English.
+This page has `lang` attribute value of `en` (English), which matches the [default language of the page][default page language]. The default language is English because all but a few words are English.
```

Comment:
```suggestion
This page has a `lang` attribute value of `en` (English), which matches the [default language of the page][default page language]. The default language is English because all but a few words are English.
```